src/performance/dashboard/src/components/modals/__tests__/ModalModifyLoadTests.test.js

```diff
@@ -0,0 +1,207 @@
+/*******************************************************************************
+* Copyright (c) 2019 IBM Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v2.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     IBM Corporation - initial API and implementation
+**/
+
+import React from 'react';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import { render, cleanup, fireEvent } from '@testing-library/react';
+
+import reducers from '../../../store/reducers/index';
+import * as DataProjectInfo from '../../../tests/data/ProjectInfo';
+import * as LoadRunnerConfig from '../../../tests/data/LoadRunnerConfig';
+import ModalModifyLoadTests from '../ModalModifyLoadTests';
+
+// initialize component props
+const componentProps = {
+    open: true,
+    closeModalWindow: jest.fn()
+}
+
+// Initialize redux stores
+const store = createStore(reducers, {
+    projectInfoReducer: { config: DataProjectInfo.projectInfo, receivedAt: Date.now(), fetched: true },
+    loadRunnerConfigReducer: { config: LoadRunnerConfig.projectLoadRunnerConfig, receivedAt: Date.now(), fetched: true }
+});
+
+// component to render
+const wrapper = (
+    <Provider store={store}>
+        <ModalModifyLoadTests {...componentProps} />
+    </Provider>
+)
+
+// Mute modal dataIconPath warnings
+console.error = () => { }
+
+// dont leak state
+afterEach(cleanup);
+
+/** 
+ * Test functionality of the ModalModifyLoadTests dialog
+ */
+describe('<ModalModifyLoadTests />', () => {
+    test('dialog renders without crashing', () => {
+        render(wrapper);
+        expect(document.querySelector('.ModalModifyLoadTest').className).toContain("bx--modal");
+    });
+
+    test('modal has a valid heading', () => {
+        render(wrapper)
+        expect(document.querySelector('.ModalModifyLoadTest .bx--modal-header__heading').innerHTML).toBe('Load test configuration');
+    });
+
+    test('there is an active cancel button', () => {
+        const { getByText } = render(wrapper);
+        const cancelButton = getByText('Cancel');
+        expect(cancelButton.disabled).toEqual(false)
+    })
+
+    test('there is a disabled submit button', () => {
+        const { getByText } = render(wrapper);
+        const submitButton = getByText('Save')
+        expect(submitButton.disabled).toEqual(true)
+    })
+
+    // Test Path Field
```

Comment:
To make these clearer when running tests, where you have these separating comments could you use new describe blocks? Just a thought
i.e 
```
describe('<ModalModifyLoadTests />', () => {

	describe('Test inputs to path field', () => {
		....
	});

	// Other describe blocks
	
});

Reply:
yes that would make the logs easier to read - made changes